client/state/ticket-support/configuration/selectors.js

```diff
@@ -0,0 +1,15 @@
+export const defaultConfiguration = {
+	isUserEligible: false,
+};
+
+export const getTicketSupportConfiguration = ( state ) => {
+	return state.ticketSupport.configuration || defaultConfiguration;
+};
```

Comment:
Other than tests, is there a situation where the entire configuration object would be needed? By using `combineReducers` as mentioned in another comment, we could do away with the `defaultConfiguration` and access the state object directly in tests.

For the individual config options (`isUserEligible` etc), it's nice to have them behind selectors which can easily be updated in one place if the configuration params change.